### PR TITLE
fixed broken table format in docs

### LINKED ISSUE #68689: Prometheus: Configuration options table format is broken in the data source documentation page
The page https://grafana.com/docs/grafana/latest/datasources/prometheus/

Image

### PR DESCRIPTION
**Pull request by krishnaduttPanchagnula**
**What is this feature?**

[Add a brief description of what the feature or update does.]

**Why do we need this feature?**

[Add a description of the problem the feature is trying to solve.]

**Who is this feature for?**

[Add information on what kind of user the feature is for.]

**Which issue(s) does this PR fix?**:

Fixes #68689

**Special notes for your reviewer:**

Please check that:
- [ ] It works as expected from a user's perspective.
- [ ] If this is a pre-GA feature, it is behind a feature toggle.
- [ ] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/#how-to-determine-if-content-belongs-in-a-whats-new-document), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/) doc.

### CODE REVIEW

**Review by jdbaldry — APPROVED** [review]
Thank you so much for contributing this fix!

I have confirmed that this renders correctly locally but we need `prettier` to be happy with the formatting before this is good to go.

I'll add the appropriate maintainer labels for you and add suggestions for the necessary formatting changes and then we can get this merged

File: docs/sources/datasources/prometheus/_index.md [hunk 1]
```diff
@@ -62,16 +62,15 @@ To configure basic settings for the data source, complete the following steps:
...
-    | **URL** | _(Visible only if you disable `Internal link`)_ Defines the external link's full URL. You can interpolate the value from the field by using the [`${__value.raw}` macro]({{< relref "../..//panels-visualizations/configure-data-links/#value-variables" >}}). |
-    | **URL label** | _(Optional)_ Adds a custom display label to override the value of the `Label name` field. |
-    | **Label name** | Adds a name for the exemplar traceID property. |
+    | ****Data source**** | *(Visible only if you enable `Internal link`)* Selects the backend tracing data store for your exemplar data. |
+    | ****URL**** | *(Visible only if you disable `Internal link`)* Defines the external link's full URL. You can interpolate the value from the field by using the [`${__value.raw}` macro]({{< relref "../..//panels-visualizations/configure-data-links/#value-variables" >}}). |
+    | ****URL label**** | *(Optional)* Adds a custom display label to override the value of the `Label name` field. |
+    | ****Label name**** | Adds a name for the exemplar traceID property. |
```
Comment (jdbaldry) [inline docs/sources/datasources/prometheus/_index.md:73]:
```suggestion
    | Name              | Description                                                                                                                                                                                                                                                    |
    | ----------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
    | **Internal link** | Enable this option if you have an internal link. When enabled, this reveals the data source selector. Select the backend tracing data store for your exemplar data.                                                                                            |
    | **Data source**   | _(Visible only if you enable `Internal link`)_ Selects the backend tracing data store for your exemplar data.                                                                                                                                                  |
    | **URL**           | _(Visible only if you disable `Internal link`)_ Defines the external link's full URL. You can interpolate the value from the field by using the [`${__value.raw}` macro]({{< relref "../..//panels-visualizations/configure-data-links/#value-variables" >}}). |
    | **URL label**     | _(Optional)_ Adds a custom display label to override the value of the `Label name` field.                                                                                                                                                                      |
    | **Label name**    | Adds a name for the exemplar traceID property.                                                                                                                                                                                                                 |
```

Comment (jdbaldry) [inline docs/sources/datasources/prometheus/_index.md:73]:
```suggestion
    | Name              | Description                                                                                                                                                                                                                                                    |
    | ----------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
    | **Internal link** | Enable this option if you have an internal link. When enabled, this reveals the data source selector. Select the backend tracing data store for your exemplar data.                                                                                            |
    | **Data source**   | _(Visible only if you enable `Internal link`)_ Selects the backend tracing data store for your exemplar data.                                                                                                                                                  |
    | **URL**           | _(Visible only if you disable `Internal link`)_ Defines the external link's full URL. You can interpolate the value from the field by using the [`${__value.raw}` macro]({{< relref "../..//panels-visualizations/configure-data-links/#value-variables" >}}). |
    | **URL label**     | _(Optional)_ Adds a custom display label to override the value of the `Label name` field.                                                                                                                                                                      |
    | **Label name**    | Adds a name for the exemplar traceID property.                                                                                                                                                                                                                 |
```

**Comment by krishnaduttPanchagnula** [comment]
added those suggestions in code and pushed another commit @jdbaldry

**Comment by jdbaldry** [comment]
Beautiful, thank you again!

**Pull request merged**